tox.ini

```
@@ -0,0 +1,17 @@
+[tox]
+envlist = py35-django22{,-user_orgs}, py38-django22{,-user_orgs}
```

I think tox don't like underscores. I'm not sure though.

```suggestion
envlist = {py35,py38}-django22-{noorgs,withorgs}
```